web-app/src/Home.js

```diff
@@ -247,13 +247,26 @@ const fetchTasks = async () => {
         setShowArchived(true);
     };
 
+    const handleLogout = () => {
+        fetch('http://localhost:8080/logout', {
```

Comment:
Does this need to be environment-specific, or will localhost work in production?

Reply:
Good catch thank you. I put in the logout constant but forgot to use it.